release-notes.md

```diff
@@ -1,4 +1,11 @@
 # Scorpius-19.13.0
+## Add action addContent to Documents service ##
```

Comment:
not critical but i think it will sound better like:
 Add addContent action to Documents service